### PR TITLE
Create evals user in cluster SSO

### PR DESCRIPTION
**Pull request by aidenkeating**

### CODE REVIEW

**Comment by aidenkeating** [comment]
@odra @pmccarthy Would you mind taking a look?

**Comment by pmccarthy** [comment]
@aidenkeating As you would say...... 👀

**Comment by odra** [comment]
code looks ok, giving it a try

File: evals/roles/rhsso/tasks/client.yml [hunk 1]
```diff
@@ -15,11 +15,15 @@
...
   when: rhsso_redirect_uri == ""
 
+- name: Retrieve RH-SSO admin user password
+  shell: oc get dc/sso -o jsonpath='{.spec.template.spec.containers[?(@.name=="sso")].env[?(@.name=="SSO_ADMIN_PASSWORD")].value}'
+  register: rhsso_admin_password_cmd
+
 - name: Generate RH-SSO auth token for admin user
   uri:
-    url: "https://{{ rhsso_route }}/auth/realms/{{ rhsso_realm }}/protocol/openid-connect/token"
+    url: "https://{{ rhsso_route }}/auth/realms/master/protocol/openid-connect/token"
```
[the remark this answers is not in the document]
Comment (odra) [reply evals/roles/rhsso/tasks/client.yml]: I believe the admin user always needs to be authenticated into the admin realm (I got 401/403 errors in the past trying to use a different realm)

[the remark this answers is not in the document]
Comment (pmccarthy) [reply evals/roles/rhsso/tasks/client.yml]: @odra @aidenkeating  Ok cool, would it be worth making this configurable all the same just in case?

**Pull request merged**